about.html

```diff
@@ -0,0 +1,185 @@
+            competition <br> from 8. June to 7 July.</p>
+        <div class="ltxtCont longTxt">
+            <p class="longTxt logolongtxt">The logo of 3D Systems Developers Summit 2022 was decided through the logo
+                competition from 8. June to 7 July.</p>
+
+            <p class="longTxt logolongtxt">After online public voting and Selection Comittee`s judging process, the logo
+                fron Joel Kano won the prize.</p>
+            <p class="logolongtxt longTxt">After consultation with Mr. Kano we optimized the prize-winning-logo for this
+                year summit as following</p>
+        </div>
+        <div class="logodisplay">
+            <div id="crLogo1">
+                <a href="index.html"><img src="img/logos/developerLogo.png" alt="3d systems logo"></a>
+
+            </div>
+        </div>
+    </section>
+    <div class="Sectionseparator"></div>
+    <section id="pastSection">
+        <h2>See the past Developers Summit</h2>
+        <div class="separator"></div>
+        <p class="logedetails">Take a look at the last two Developers Summits which took place in California and Bombay
+        </p>
+        <div class="eventPicCont">
+            <div class="eventPic">
+                <span class="dateLast">2016 <br> <span class="dateLastTxt"> Developers Summit 2016 in California
+                    </span></span>
+            </div>.
+            <div class="eventPic2">
+                <span class="dateLast">2018 <br> <span class="dateLastTxt"> Developers Summit 2018 in
+                        Mombay</span></span>
+            </div>
+        </div>
+    </section>
+    <section id="apartners">
+        <h2 id='partnersh2'>Partner</h2>
+        <div id='partnersimgcont'>
+            <img class='partnersimg' src="img/logos/mozilla.png" alt="mozilla logo">
+            <img class='partnersimg' src="img/logos/airbnb.png" alt="air bnd logo">
+            <img src="img/logos/g.png" class='partnersimg' alt="google logo">
+            <img src="img/logos/naver.png" alt="naver logo" class='partnersimg'>
+            <img src="img/logos/daumkakao.png" alt="" class='partnersimg'>
+        </div>
+    </section>
+    <div class="hideAll">
+        <footer id="copyRight">
+           
+            <div id="crLogo">
+              <a href="index.html"><img id="crLogoimg" src="img/logos/icon.png" alt="3d systems logo"></a>
+              <p  class="crTxtLogo">Developers<br>Summit 2022</p>
+            </div>
+          
+            <div id="crTxt">
+              <h3 class="crtTxt">3D Systems 2022</h3>
+              <p class="crtTxt hideAll">freedom-respecting software</p>
+              <p class="crtTxt hideText">freedom-respecting software Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Id velit ut tortor pretium viverra suspendisse potenti nullam ac. Leo in vitae turpis massa.</p>
```

Comment:
- [OPTIONAL] Please avoid lines of code that are too long (100 characters or more). Makes your code cleaner and more readable. Follow the [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md). As not required by the previous reviewer I consider this optional.